### PR TITLE
Add a regression test case for runner stopping on the first failure

### LINKED ISSUE #116: Add a regression test case for runner stopping on the first failure
Fixed here - f862d92479048ce5385bec906dd7e64227b07235

I didn't add a regression test case

### PR DESCRIPTION
**Pull request by svtkachenko**
Closes #116

### CODE REVIEW

File: test/test_runner.py [hunk 1]
```diff
@@ -75,7 +82,7 @@ def assert_not_request(app, method, path):
 def test_execute(server, app):
     headers = {"Authorization": "Bearer 123"}
     execute(f"http://127.0.0.1:{server['port']}/swagger.yaml", api_options=dict(headers=headers))
-    assert len(app["saved_requests"]) == 2
+    assert len(app["saved_requests"]) >= 2
```
Comment (Stranger6667) [inline test/test_runner.py]: How many exactly?

Comment (svtkachenko) [reply test/test_runner.py]: 3

Comment (Stranger6667) [reply test/test_runner.py]: Please, set the exact number

Comment (svtkachenko) [reply test/test_runner.py]: Next time `app` would be expanded it would affect these numbers in each test in this module :(

Comment (Stranger6667) [reply test/test_runner.py]: At least it will be explicit

File: test/test_runner.py [hunk 2]
```diff
@@ -88,19 +95,19 @@ def test_execute_base_url(server, app):
...
     app["config"]["raise_exception"] = True
     stats = execute(f"http://127.0.0.1:{server['port']}/swagger.yaml")
     assert "not_a_server_error" in stats.data
-    assert dict(stats.data["not_a_server_error"]) == {"total": 3, "ok": 1, "error": 2}
+    assert dict(stats.data["not_a_server_error"]) == {"total": 5, "ok": 1, "error": 4}
 
 
 def test_auth(server, app):
     execute(f"http://127.0.0.1:{server['port']}/swagger.yaml", api_options=dict(auth=("test", "test")))
-    assert len(app["saved_requests"]) == 2
```
Comment (Stranger6667) [inline test/test_runner.py]: Could you, please, set the exact number on this line as well?

File: test/test_runner.py [hunk 3]
```diff
@@ -88,19 +95,19 @@ def test_execute_base_url(server, app):
...
+    assert len(app["saved_requests"]) >= 2
```
Comment (Stranger6667) [inline test/test_runner.py]: Could you, please, set the exact number on this line as well?

**Pull request merged**